mindocr/models/heads/cls_mv3_head.py

```diff
@@ -0,0 +1,27 @@
+import mindspore.nn as nn
+from mindspore import Tensor
+
+from ..backbones.mindcv_models.layers.pooling import GlobalAvgPooling
+
+
+class ClsHead(nn.Cell):
+    """
+    Text direction classification head.
+    """
+    def __init__(self, in_channels: int, out_channels: int, num_classes: int):
+        super().__init__()
+        self.pool = GlobalAvgPooling()
+        self.classifier = nn.SequentialCell([
+            nn.Dense(in_channels, out_channels),
```

Comment:
`out_channels` usually corresponds to the real output channels (or the `num_classes` in this case ), it is better to name it as "hidden_channels"

Reply:
updated